oarepo_ui/templates/components/IdentifiersAndLinks.jinja

```diff
@@ -1,9 +1,23 @@
-{# def originalRecordUrl=None, objectIdentifiers=None #}
-
-{% if objectIdentifiers|length > 0 or originalRecordUrl %}
+{# def originalRecordUrl=None, objectIdentifiers=None, repositoryAssignedIdentifier=None  #}
+{% if objectIdentifiers|length > 0 or originalRecordUrl or repositoryAssignedIdentifier %}
```

Comment:
accept pids and render all pids here, to make component more universal. For now we have only single DOI as managed PID, for other repos there may be more